tests/usm/usm_api_copy_handler_no_events.cpp

```diff
@@ -0,0 +1,52 @@
+/*******************************************************************************
+//
+//  SYCL 2020 Conformance Test Suite
+//
+//  Provides tests for overload of the handler::copy() method with no events
```

Comment:
@keryell Just out of curiosity: isn't it a synonym? Looks like even C++ standard uses this term sometimes:

> [string.view.iterators]
For a basic_string_view str, any operation that invalidates a pointer in the range [str.data(),
str.data() + str.size()) invalidates pointers, iterators, and references returned from str’s methods.

Reply:
I've replaced "method" to "member function" in this commit: 800816219c8f8b2a87887df4ef54c8088b0a153b

Reply:
@yuriykoch yes, you are right is has the same meaning, but since SYCL is a standard based on C++, another standard, we have the great responsibility to be precise. :-)
The example you cite has been fixed with https://github.com/cplusplus/draft/pull/1574 fixing the problem I mention: https://github.com/cplusplus/draft/issues/1573
I search the latest version of the C++ draft standard for `method` and there is only remaining use-cases with the usual English meaning.